lib/markdown.ts

```diff
@@ -119,36 +119,36 @@ export class DocsWorkspace implements IWorkspace {
   }
 
   hasMarkdownDocument(resource: URI) {
-    const relativePath = path.relative(URI.file(this.root).path, resource.path);
+    const relativePath = path.relative(path.resolve(this.root), resource.fsPath);
```

Comment:
One change I made here was to just use `path.resolve` to get an absolute path for the root dir. I think that's easier to reason about than `URI`'s handling of relative paths, but I'm happy to switch this to `URI.file(this.root).fsPath` instead if that's preferred.